dipy/segment/metric.py

```diff
@@ -1,15 +1,23 @@
 from dipy.segment.featurespeed import (Feature,
                                        IdentityFeature,
-                                       CenterOfMassFeature)
+                                       ResampleFeature,
+                                       CenterOfMassFeature,
+                                       MidpointFeature,
+                                       ArcLengthFeature,
+                                       VectorBetweenEndpointsFeature)
 
 from dipy.segment.metricspeed import (Metric,
                                       SumPointwiseEuclideanMetric,
                                       AveragePointwiseEuclideanMetric,
-                                      MinimumAverageDirectFlipMetric)
+                                      MinimumAverageDirectFlipMetric,
+                                      CosineMetric)
 
 from dipy.segment.metricspeed import (dist,
                                       distance_matrix)
 
+# Creates aliases
+EuclideanMetric = SumPointwiseEuclideanMetric
```

Comment:
What you said makes sense, I'll create a new class that subclass `SumPointWiseEuclideanMetric`. Thanks for the idea.